microservices/product-search/hooks/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
After you build your microservice on dockerhub or whatever? How does it get into project-base configuration or? How do you even use it? I feel like maybe building it using our CI could be much cleaner. Because right after build you can set it into kubernetes manifests. This way, I will need to wait to split the monorepo, then wait for build to finish and after that somehow set it to project-base docker-compose right? 

I think that you should upgrade release checklist, or come up with some solution, because at this time im not really sure how these images get into project-base

Reply:
Also this way, If Im deploying my changes in monorepo on our Circle i got no chance to get this Image right? because it does not exist. This is something that we must find out how to do. 

My personal opinion is that we should build these images by ourselves and always set it into manifests manually.

Then there should be process while releasing new version of shopsys framework to build images with some specific version(maybe tag in the git) and set it into docker-composes in project-base.


I would really like to hear your opinion on how this will work for us.